test/issues/940.test.js

```diff
@@ -160,7 +160,7 @@ describe("#940", () => {
       await promise2;
 
       background.browser.tabs.create.should.have.been.calledTwice;
-    }).timeout(2002);
```

Comment:
Why this change out of interest?

Reply:
This test was passing before the refactor and began failing after making the changes. As the test seems unrelated to the changes, I increased the timeout. (Two millis is the minimum increase necessary to make the test pass on my machine.)

Reply:
The test is passing on CI after reverting the timeout change 🤷‍♂

Reply:
Heh thanks!